src/lib/styles/global/link.scss

```diff
@@ -16,7 +16,13 @@ a {
   }
 
   &.button {
-    @include button.base;
+    &.primary,
```

Comment:
Why not using `.button:not(.text)` instead?

Reply:
To follow the same structure as in `buttons.scss` [@include button.base;](https://github.com/dfinity/gix-components/blob/f5dcbb37797595af47afde13e15476670e0e6902/src/lib/styles/global/button.scss#L61)

Reply:
It makes sense to be consistent. Thanks!